### DISCUSSION TITLE
Welcome to zalgebra Discussions!

### DISCUSSION THREAD

**Discussion by kooparse** [kooparse/zalgebra#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by Rhinemann** [comment]
Hello. I have a question about the library. Specifically, how would you pass matrix data from 2d array to OpenGL? Since the library is meant for games. (I'm rather new to Zig.)

**Comment by Rhinemann** [comment]
Also, another question, out of curiosity, are Mat4-s 2d arrays for convenience reasons instead of vectors or is it something different?
